src/Utils.ts

```diff
@@ -9,23 +9,17 @@ export function mapEvent<I, O>(event: Event<I>, map: (i: I) => O): Event<O> {
     return (listener, thisArgs = null, disposables?) => event(i => listener.call(thisArgs, map(i)), null, disposables);
 }
 
-export namespace Utils
-{
-    export function normalizePath(path: string): string
-    {
-        var normalizedPath = path;
-
-        if (!pathIsUNC(normalizedPath)) {
-            var replaceable = normalizedPath.split('\\');
-            normalizedPath = replaceable.join('\\\\');
+export namespace Utils {
+    // normalize function for turning windows paths into
```

Comment:
This will break mac and linux compatibility

Reply:
I moved the UNC processing to where it was being used, since it's only used once, getting the p4 command location (and already platform specific), having both of these with the similar name was easily confused, and the conversion from \ to / should work with all plats.   The main need for this is due to a bug in vscode (reported, now fixed) where asRelativePath returns a normalized path for not found, instead of the provided input.  Normalizing everything before comparing is hitting it with a hammer, but it seems to work -- no *nix box to test, though, so here's to hoping...